src/libraries/System.Net.Mail/ref/System.Net.Mail.cs

```diff
@@ -8,12 +8,12 @@ namespace System.Net.Mail
 {
     public partial class AlternateView : System.Net.Mail.AttachmentBase
     {
-        public AlternateView(System.IO.Stream contentStream) : base (default(System.IO.Stream)) { }
-        public AlternateView(System.IO.Stream contentStream, System.Net.Mime.ContentType? contentType) : base (default(System.IO.Stream)) { }
-        public AlternateView(System.IO.Stream contentStream, string? mediaType) : base (default(System.IO.Stream)) { }
-        public AlternateView(string fileName) : base (default(System.IO.Stream)) { }
-        public AlternateView(string fileName, System.Net.Mime.ContentType? contentType) : base (default(System.IO.Stream)) { }
-        public AlternateView(string fileName, string? mediaType) : base (default(System.IO.Stream)) { }
+        public AlternateView(System.IO.Stream contentStream) : base (default(string)) { }
```

Comment:
I don't know if it matters here, but this calls to a different base constructor than it used to and than what the actual source code calls. Is that OK (i.e. non-breaking)?

Reply:
> I don't know if the ref assembly matters as long as it shows correct constructor prototypes

It does not matter. The base constructor call is there just to make C# compiler happy.

It looks like a non-determinism in the auto-generator. It used to call `string` overload, then it flipped to Stream overload in https://github.com/dotnet/corefx/pull/35557/, and it is flipping back to string overload again.

Reply:
As @jkotas mentioned the base constructor call is only required to make it compile.  The one chosen is based on a heuristic: https://github.com/dotnet/arcade/blob/e375d9151b2345e26a0bf967913fa4d621891496/src/Microsoft.Cci.Extensions/Writers/CSharp/CSDeclarationWriter.Methods.cs#L429

Typically those are stable based on metadata order, or they are ordinal sorted based on the signature.  I could imagine a problem where GenAPI might have been missing references, and sorted `System.IO.Stream` before `System.String` - that could have happened when https://github.com/dotnet/corefx/pull/35557 was submitted.

If you notice this changing a lot on rerun of the tool in the current codebase, that'd be a bug to fix.  So long as it's stable now I'd just ignore this change and accept it to align with the current output.